src/modules/emote_menu/components/EmoteMenu.jsx

```diff
@@ -34,6 +34,10 @@ export default function EmoteMenu({toggleWhisper, appendToChat, onSetTip}) {
         return;
       }
 
+      if (emote.metadata && emote.metadata.isLocked()) {
```

Comment:
if it's locked, why are you able to click on it or hover it when using keyboard shortcuts. ideally clicking on locked elements should not result in a click event unless you're undoing some action, like unfavoriting.